app/src/main/java/org/wikipedia/activity/BaseActivity.kt

```diff
@@ -90,6 +97,15 @@ abstract class BaseActivity : AppCompatActivity() {
         maybeShowLoggedOutInBackgroundDialog()
 
         Prefs.localClassName = localClassName
+
+        val decorView = window.decorView
+        decorView.viewTreeObserver.addOnGlobalLayoutListener { ViewUtil.setTouchListenersToViews(window.decorView, this) }
```

Comment:
Please use `decorView` in the `addOnGlobalLayoutListener` instead of `window.decorView`.